clientd3d/annotate.c

```diff
@@ -98,11 +98,11 @@ void MapMoveAnnotations( MapAnnotation *annotations, int x, int y, float scale,
        continue;
 
      // Set up tooltip for annotation
-     ti.rect.left = view.x + x + (int) ((annotations[i].x - MAP_ANNOTATION_SIZE / 2) * scale);
```

Comment:
I realized just now that this does not take into account hitting the maximum annotation size, so the tooltip hotspot zone is larger than the capped size of the annotation graphic. I'll follow up once I figure this out.

Reply:
I'm stumped by this one even though the solution seems obvious. When `MapMoveAnnotations()` draws the annotation tooltip control, it needs to do so using the scaled radius now used to draw the annotation. I figured this was just a matter of passing this calculated radius to `MapMoveAnnotations()` and using that, but after various attempts I've come up empty-handed. I'll keep at this of course, but will gladly accept advice.

Reply:
Managed to figure it out in the latest commits. I've updated the function signature to support passing in the scaled radius rather than calculate it again within `MapMoveAnnotations()`.